common/source/docs/common-realsense-depth-camera.rst

```diff
@@ -0,0 +1,249 @@
+
+
+Configure companion computer
+============================
+
+Install ``librealsense`` and ``pyrealsense2``
+---------------------------------------------
+
+The Realsense depth camera is supported via `librealsense <https://github.com/IntelRealSense/librealsense>`__ on Windows and Linux. Installation process varies widely for different systems, hence refer to `the official github page <https://github.com/IntelRealSense/librealsense>`__ for instructions for your specific system:
+
+- `Ubuntu <https://github.com/IntelRealSense/librealsense/blob/master/doc/installation.md>`__
+- `Jetson <https://github.com/IntelRealSense/librealsense/blob/master/doc/installation_jetson.md>`__
+- `Odroid <https://github.com/IntelRealSense/librealsense/blob/master/doc/installation_odroid.md>`__
+- `Windows <https://github.com/IntelRealSense/librealsense/blob/master/doc/installation_windows.md>`__
+- `Raspbian <https://github.com/IntelRealSense/librealsense/blob/master/doc/installation_raspbian.md>`__
+
+Establish serial connection with ArduPilot
+------------------------------------------
+
+You can follow the wiki to :ref:`connect to ArduPilot with MAVLink <raspberry-pi-via-mavlink>`.
+
+Install supporting packages
+---------------------------
+
+First install `Python3 for Ubuntu <https://realpython.com/installing-python/#ubuntu>`__ (not necessary for Ubuntu 18.04 and above). You should be able to then run the examples provided by Intel can be found in the folder ``~/librealsense/wrappers/python/examples`` with Python3 command.
+
+.. code-block:: bash
+
+    # Update the PYTHONPATH environment variable to add the path to the pyrealsense2 library
+    export PYTHONPATH=$PYTHONPATH:/usr/local/lib
+
+    cd ~/librealsense/wrappers/python/examples
+
+    # You should see a stream of depth data coming from the D4xx camera.
+    python3 python-tutorial-1-depth.py
+
+Install pip for Python3 `(pip3) <https://linuxize.com/post/how-to-install-pip-on-ubuntu-18.04/#installing-pip-for-python-3>`__ and other supporting packages:
+
+.. code-block:: bash
+
+    sudo apt-get install python3-pip
+    pip3 install pyrealsense2
+    pip3 install transformations
+    pip3 install dronekit
+    pip3 install apscheduler
+    pip3 install pyserial # For serial connection
+    pip3 install numba
+    pip3 install opencv-python
+    sudo apt -y install python3-gst-1.0 gir1.2-gst-rtsp-server-1.0 gstreamer1.0-plugins-base gstreamer1.0-plugins-ugly libx264-dev
+
+    # Only necessary if you installed the minimal version of Ubuntu
+    sudo apt install python3-opencv
+
+
+Download the main script `d4xx_to_mavlink.py <https://github.com/thien94/vision_to_mavros/blob/master/scripts/d4xx_to_mavlink.py>`__ or clone the `vision_to_mavros <https://github.com/thien94/vision_to_mavros>`__ repository and find the script folder.
+
+.. code-block:: bash
+
+    # Clone the package in the ROS workspace or any location if you don't use ROS
+    cd ~/catkin_ws/src
```

Comment:
not using ROS so do we really want to put these scripts in the catkin directory?

Reply:
@rmackay9
I will change it to be arbitrary then, with the note that for ROS-user they can place it in the catkin directory.